index.js

```diff
@@ -306,6 +306,11 @@
     };
   };
 
+  var ValidDate = $.NullaryType(
+    'ValidDate',
+    function (x) { return $$typeEq('Date')(x) && !isNaN(x.getTime()) }
```

Comment:
That's a solid argument in favour of `.getTime()`. Since we used `.valueOf()` in #12, though, I'd prefer to use it here as well.